build/yamls/ipfix-collector.yml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app: ipfix-collector
+  name: ipfix
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ipfix-collector
+  name: ipfix-collector
+  namespace: ipfix
+spec:
+  selector:
+    app: ipfix-collector
+  ports:
+    - protocol: TCP
```

Comment:
We support UDP too. 
https://github.com/vmware/go-ipfix/blob/main/cmd/collector/collector.go#L60

Reply:
Do we have to add a configmap to support UDP?

Reply:
Do we have to add a configmap to support UDP?

Reply:
Command argument `--ipfix.transport="udp"` in the k8s manifest should work.

Reply:
It looks like in order to make it work, I have to modify the Dockerfile.build.collector. 
Line19 is changed from:
`ENTRYPOINT "collector"`
to:
`ENTRYPOINT ["collector"]`
And now collector.go can effectively take arguments.

Reply:
Thanks for trying it out and figuring out the fix.